### ISSUE TITLE
Importing .excalidraw from Web Browser Shows Blank File

### ISSUE THREAD

**Issue by Snuggle** [zsviczian/obsidian-excalidraw-plugin#56]
#  Importing .excalidraw from Web Browser Shows Blank File
## **Describe the bug**
Hello! Whenever I make an excalidraw file in my web browser and try to add it to my Obsidian vault, this plugin shows a blank empty canvas and then overwrites the file with a blank empty canvas.

### Steps to reproduce the behavior:
1. Create a blank canvas from scratch with Excalidraw
2. Draw something! (Anything)
3. Export the `.excalidraw` file.
4. Move the exported `something.excalidraw` file into your Obsidian vault
5. Open the file with this plugin
6. See results

### **Video of how to Reproduce this Bug**

[![image](https://user-images.githubusercontent.com/26250962/117892657-04896200-b2b1-11eb-977a-561fe99ce621.png)](https://www.youtube.com/watch?v=35KrC6M2WFI)
## **Expected behavior**
I wanted to be able to view and edit my file within Obsidian, but instead it's overwritten/deleted with a blank canvas. It seems this plugin doesn't support it.

## **Desktop (please complete the following information):**
 - OS: Ubuntu 21.04 [e.g. iOS]
 - Browser: Firefox [e.g. chrome, safari]
 - Broswer Version: 88.0 [e.g. 22]

### **Obsidian Version (v0.12.3 - Up-to-date)**
![image](https://user-images.githubusercontent.com/26250962/117892768-37cbf100-b2b1-11eb-8620-da37aab825a8.png)

###**Plugin Version (v1.0.11 - Up-to-date)**
![image](https://user-images.githubusercontent.com/26250962/117893127-ba54b080-b2b1-11eb-9dbc-95b57725438b.png)

**Comment by zsviczian** [comment]
Thanks for the very well documented issue! I was able to reproduce it based on your description.

The issue seems to be connected to freehand drawing only. i.e. if you only add boxes, circles, arrows, text... and save your Excalidraw file, those will import well. Freehand drawings in excalidraw.com indeed don't work in obsidian. I will investigate why. It seems as if there is a version difference between excalidraw.com component and the officially published excalidraw component that I am using in the Obsidian solution. The difference could be related to support for pressure sensitive drawing on excalidraw.com.

I tried free hand drawing in Obsidian, copying that to Excalidraw.com, then saving it as an .excalidraw file and importing to Obsidian. That also works.

long story short... I'll get to the bottom of this, but may take a few days as I am not yet clear what exactly the issue is.

**Comment by zsviczian** [comment]
Thanks for raising this. I deployed Excalidraw 0.8.0 to Obsidian. Please update to 1.0.12, and it will work.

**Comment by Snuggle** [comment]
@zsviczian Thank you so much for fixing this and I hope you have a wonderful weekend! :blush:
